Salary App/Features/ValidationManager/ValidationManager.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ValidationManager.swift
+//  Salary App
+//
+//  Created by Tobi Adegoroye on 12/07/2023.
+//
+
+import Foundation
+
+enum FormError: Error {
+    case containsEmptyString
+    case formContainsLetters
+    case invalidSalary
+    case isValidPenContribution
+}
+
+extension FormError: LocalizedError {
+    var errorLocalDescription: String {
+        switch self {
+        case .containsEmptyString:
+            return "The salary textfield contains an empty string, please enter a value"
+        case .formContainsLetters:
+            return "This textfield contains letters, please enter a number"
+        case .invalidSalary:
+            return "This is not a valid salary"
+        case .isValidPenContribution:
+            return "This is not a valid pension contribution"
+        }
+    }
+}
+
+struct ValidationManager {
```

Comment:
This should only have one function that accepts a struct where you validate all its properties and then throw an error when a condition isn't met.